src/utils/log.js

```diff
@@ -2,17 +2,14 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-// import mozlog from 'mozlog'
+import mozlog from 'mozlog'
 
 import AppConstants from '../appConstants.js'
 
-// mozlog temporarily disabled because its module system appears to not work well
-// with Next.js (uncomment this and comment out line 16 to see the errors)
-// const log = mozlog({
-//   app: 'fx-monitor',
-//   level: AppConstants.MOZLOG_LEVEL,
-//   fmt: AppConstants.MOZLOG_FMT
-// })
-const log = () => console
+const log = mozlog({
+  app: 'fx-monitor',
+  level: AppConstants.MOZLOG_LEVEL,
+  fmt: AppConstants.MOZLOG_FMT
+})
 
-export default log
+export default log
```

Comment:
nit: looks like a missing newline at the end of the file. 

https://eslint.org/docs/latest/rules/eol-last

Reply:
Fixed while rebasing.